my_portfolio/resources/css/style.css

```diff
@@ -0,0 +1,733 @@
+  top: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 1;
+  align-items: center;
+  text-align: center;
+}
+
+.dot.blue {
+  background-color: #0c07f2;
+  padding: 4px;
+  margin-bottom: 23px;
+}
+
+#contact-box {
+  background-color: #d3d2ff;
+  max-width: 300px;
+  padding: 10px 15px;
+  position: relative;
+  margin: auto;
+  border-radius: 8px;
+}
+
+#contact {
+  padding-bottom: 60px;
+}
+
+input {
+  border: none;
+  padding: 10px 20px;
+  width: 250px;
+}
+
+textarea {
+  border: none;
+  padding: 7px 20px;
+  width: 250px;
+  height: 120px;
+}
+
+.txt {
+  margin-top: 20px;
+}
+
+label {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 30px;
+}
+
+button {
+  align-self: flex-end;
+  background-color: #0c07f2;
+  color: #fff;
+  padding: 8px 50px;
+  border: none;
+  cursor: pointer;
+  margin-right: 2vh;
+  margin-top: 30px;
+  margin-bottom: 20px;
+  box-shadow: 0 8px 10px 1px rgba(0, 0, 0, 0.15);
+  border-radius: 5px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.wrap {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.skills ul {
+  display: inline-flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: 15px;
+  margin-bottom: 0;
+}
+
+.skills li {
+  list-style: none;
+  font-weight: 600;
+  padding: 5px 15px;
+  background-color: #f4f4f4;
+}
+
+.about_contents {
+  display: flex;
+  padding: 10px 50px;
+  gap: 20px;
+  justify-content: space-between;
+  padding-bottom: 40px;
+  flex-direction: column;
+}
+
+.about_contents h3 {
+  margin-bottom: 20px;
+}
+
+.skills {
+  max-width: 350px;
+}
+
+@media screen and (min-width: 768px) {
+  .menu-btn {
+    visibility: hidden;
+  }
+
+  header img {
+    width: 45px;
+    border-radius: 100px;
+  }
+
+  .nav {
+    visibility: visible;
+    background-color: #fff;
+    height: 8.6vh;
+    z-index: 2;
+    position: fixed;
+  }
+
+  .nav .menu-nav {
+    display: block;
+    transform: translateY(0);
+    background: transparent;
+    text-align: right;
+    margin-right: 85px;
+    width: auto;
+    height: auto;
+    margin-top: auto;
+    position: relative;
+    z-index: 0;
+    right: -25px;
+  }
+
+  .nav .menu-nav_item {
+    display: inline;
+    padding-right: 2vw;
+  }
+
+  .nav .menu-nav_link {
+    font-weight: 700;
+    font-size: 15px;
+    padding: 1rem 0;
+    display: inline-block;
+    margin-left: 2.3vw;
+  }
+
+  .nav .menu-nav_link:hover {
+    border-bottom: 2px solid#AD00FF;
+  }
+
+  .name {
+    margin-left: 85px;
+  }
+
+  .footer .social-icons {
+    flex-direction: row;
+  }
+
+  .text-secondary {
+    font-size: 35px;
+  }
+
+  #home .social-icons {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    background-color: #fff;
+    right: 0;
+    justify-content: center;
+  }
+
+  .footer .social-icons i {
+    font-size: 25px;
+    margin-left: 20px;
+  }
+
+  #home .social-icons i {
+    font-size: 25px;
+  }
+
+  #home {
+    height: 100vh;
+  }
+
```

Comment:
- this is a duplicate no need move the style into the first ```#home``` and delete
- Pls check there a couple and as you can there ```home``` rules down here pls make them be in one section for harmony and consistency

Reply:
Noted sir